recipes/fastjet-contrib/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set name = "fastjet-contrib" %}
+{% set version = "1.056" %}
+
+package:
+  # Have top level name be unique from any outputs
+  # c.f. https://github.com/conda-forge/conda-forge.github.io/blob/abfc33db28c35e9a8f6b719d0021768f0d5d06be/docs/maintainer/knowledge_base.md?plain=1#L1749
+  name: {{ name }}-split
+  version: {{ version }}
+
+source:
+  url: https://fastjet.hepforge.org/contrib/downloads/fjcontrib-{{ version }}.tar.gz
+  sha256: fc31544424dece0d0676ea2433ad1e96fd9db82920bc7a7ef6294ce94e659d6e
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}
+
+    build:
+      skip: true  # [win]
+      run_exports:
+        - {{ pin_subpackage('fastjet-contrib', max_pin='x.x') }}
+      script:
+        # Get an updated config.sub and config.guess
+        - cp $BUILD_PREFIX/share/gnuconfig/config.* .
+        - ./configure --help
+        - ./configure --list
+        - ./configure --fastjet-config=$(command -v fastjet-config) --prefix=$PREFIX CXX=$CXX
+        - make --jobs="${CPU_COUNT}"
+        - make fragile-shared --jobs="${CPU_COUNT}"
+        - make check --jobs="${CPU_COUNT}"  # [build_platform == target_platform]
+        - make install
+        - make fragile-shared-install
+        # Remove the static libraries that got installed
+        - rm $PREFIX/lib/libCentauro.a
+        - rm $PREFIX/lib/libClusteringVetoPlugin.a
+        - rm $PREFIX/lib/libConstituentSubtractor.a
+        - rm $PREFIX/lib/libEnergyCorrelator.a
+        - rm $PREFIX/lib/libFlavorCone.a
+        - rm $PREFIX/lib/libGenericSubtractor.a
+        - rm $PREFIX/lib/libJetCleanser.a
+        - rm $PREFIX/lib/libJetFFMoments.a
+        - rm $PREFIX/lib/libJetsWithoutJets.a
+        - rm $PREFIX/lib/libLundPlane.a
+        - rm $PREFIX/lib/libNsubjettiness.a
+        - rm $PREFIX/lib/libQCDAwarePlugin.a
+        - rm $PREFIX/lib/libRecursiveTools.a
+        - rm $PREFIX/lib/libScJet.a
+        - rm $PREFIX/lib/libSignalFreeBackgroundEstimator.a
+        - rm $PREFIX/lib/libSoftKiller.a
+        - rm $PREFIX/lib/libSubjetCounting.a
+        - rm $PREFIX/lib/libValenciaPlugin.a
+        - rm $PREFIX/lib/libVariableR.a
+
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('cxx') }}
+        - automake
+        - libtool
+        - make
+        - pkg-config
+        - gnuconfig
+        - fastjet-cxx  # [build_platform != target_platform]
+      host:
+        - fastjet-cxx
+      run:
+        - fastjet-cxx
+
+    test:
+      requires:
+        - {{ compiler('cxx') }}
+        - curl
+        - tar
+        - grep
+        - sed
+      script: run_test_shared.sh
+
+  - name: {{ name }}-static
```

Comment:
Question for reviewers: From [CFEP 18](https://github.com/conda-forge/cfep/blob/22ff773dea82854154c181435161eaa03e608fa1/cfep-18.md#implementation)

> For now, we will also don't do any automatic rebuilds for packages with static linkage. This means that packages using static linkage need to regularly rebuild to include the updated versions. Otherwise the end-user experience will be that the library wasn't updated at all in the stack.

so if I understand it correctly, because this package has a static output it will never get automatic rerenders?

Reply:
It will, the only thing we don't support is correct rebuild on downstreams that use static libraries.

Reply:
Thanks very much @xhochy!